keps/sig-apps/4212-declarative-node-maintenance/kep.yaml

```diff
@@ -0,0 +1,48 @@
+title: Declarative Node Maintenance
+kep-number: 4212
+authors:
+  - "@atiratree"
+owning-sig: sig-apps
+participating-sigs:
+  - sig-cli
+  - sig-apps
+  - sig-node
+  - sig-scheduling
+status: provisional
+creation-date: 2023-09-15
+reviewers:
+  - @sftim
+  - @fabiand
+  - @mmerkes
+  - @rptaylor
+  - @pwschuurman
```

Comment:
Seems like these should be quoted, according to [tests](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/enhancements/4213/pull-enhancements-test/1754855493743939584#1:build-log.txt%3A964).
> 4212-declarative-node-maintenance/kep.yaml:[parsing KEP file: unmarshalling YAML: yaml: line 14: found character that cannot start any token]]" should have 0 item(s), but has 1

Reply:
thanks - should be better now